app/presenters/result_set_presenter.rb

```diff
@@ -80,7 +80,7 @@ def sort_options
 
     sort_option ||= finder.default_sort_option
 
-    "sorted by <strong>" + sort_option['name'] + "</strong>" if sort_option.present?
+    "<span class='visually-hidden'>, sorted by <strong>" + sort_option['name'] + "</strong></span>" if sort_option.present?
```

Comment:
Is there a reason why this starts with a comma?

Reply:
Probably only because it had one before. I'll check it with a screenreader to see if it provides a useful pause or not, but otherwise yeah, could be removed, thanks.

Reply:
I can't decide so I've taken it out. Better not to second guess these things.